core/project.clj

```diff
@@ -27,7 +27,8 @@
                                   [com.bhauman/figwheel-main "0.2.13"]
                                   [org.clojure/tools.reader "1.3.5"]
                                   [cider/piggieback "0.5.2"]
-                                  [org.clojure/tools.nrepl "0.2.13"]]
+                                  [org.clojure/tools.nrepl "0.2.13"]
+                                  [nubank/matcher-combinators "3.8.5"]]
```

Comment:
I've added this library that helps to write more concise assertions (and also provides a colored diff when assertions fail to spot easily the difference btw expected and actual), but let me know if you prefer me to remove it.